geode-junit/src/main/java/org/apache/geode/test/junit/assertions/TabularResultModelColumnAssert.java

```diff
@@ -0,0 +1,39 @@
+package org.apache.geode.test.junit.assertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.List;
+
+public class TabularResultModelColumnAssert<T> extends TabularResultModelSliceAssert<T> {
```

Comment:
I am not sure of this. we used ListAssert before so that we don't have duplicate all those methods in our own code, now it looks like we are.

Reply:
the difference is a ListAssert is a dead-end (ListAssert methods return ListAssert, so you cannot get to the next column by chaining).  These methods return TabularResultModelAssert, so you can continue easily to the next column.  Duplicating a few common methods is the price of clean test code, otherwise tests must be broken up with more temporary variables.

I also find it cleaner to separate the row/column selection from the subsequent validation.  Before, we had some methods that combines these two steps, which would have resulted in 3x as many permutation methods to achieve equivalent flexibility.

Reply:
Is there a way to extend the ListAssert and then have a method to go back up to TabularResultModelAssert?

Reply:
@Petahhh and I spent a day exploring all possible ways to use ListAssert more directly.  None look as clean as what's provided in this PR.  It is not possible to extend ListAssert to add a method to get back up, because all ListAssert methods' return type is still ListAssert, not the subclass type.  I did add an asList() method to provide access to full ListAssert capabilities when desired, and documented the tradeoff that using asList() means you will not be able to chain additional row or column validations.